test/ActiveLogin.Identity.Swedish.Test/coverage.xml

```diff
@@ -0,0 +1,3173 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
Should this file really be in source control, is it not the output of running the alt-cover-tool?

Reply:
Totally correct :) Removed and added to .gitignore. Thanks for heads up!